### PR TITLE
fix: google analytics tags

### PR DESCRIPTION
**Pull request by imsitnikov**

### CODE REVIEW

File: src/pages/_document.tsx [hunk 1]
```diff
@@ -1,14 +1,43 @@
...
 import block from 'bem-cn-lite';
 import {Head, Html, Main, NextScript} from 'next/document';
+import Script from 'next/script';
+
+import {IS_PRODUCTION} from '../constants';
+
 const theme = Theme.Dark;
 const b = block('yc-root');
 
+const GA_ID = 'GTM-KHT6KD7';
```
Comment (melikhov-dev) [inline src/pages/_document.tsx:11]: From my points of view `config.ts` or `constants.ts` is the better place for this

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,66 +1,11 @@
...
-
-                <meta property="share:title" content="Gravity&nbsp;UI" />
-                <meta property="share:sharing_schema" content="default" />
-                <link rel="manifest" href="/manifest.json" />
-            </Head>
-
-            <Layout>
-                <Landing />
-            </Layout>
-        </>
```
Comment (melikhov-dev) [inline src/pages/index.tsx:63]: Why you delete all meta's?

File: src/pages/_document.tsx (hunk 1 above)
Comment (imsitnikov) [reply src/pages/_document.tsx]: done

File: src/pages/index.tsx (hunk 2 above)
Comment (imsitnikov) [reply src/pages/index.tsx:63]: This was duplication of meta from `Layout`

**Review by melikhov-dev — APPROVED** [review]

**Pull request merged**